release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -872,6 +872,11 @@ With this release, maximum transmission unit (MTU) on virtual function using the
 
 For more information, see xref:../networking/hardware_networks/add-pod.adoc#nw-sriov-expose-mtu_configuring-sr-iov[Exposing MTU for vfio-pci SR-IOV devices to pod].
 
+[id="ocp-4-17-nw-metallb-metric-labels"]
+==== MetalLB metrics naming update
+
+With this release, the naming convention for MetalLB BGP and BFD metrics is updated. For more information, see xref:../networking/metallb/metallb-troubleshoot-support.adoc#nw-metallb-metrics_metallb-troubleshoot-support[MetalLB metrics for BGP and BFD].
```

Comment:
Also, maybe it is technically correct, but "is updated" sounded weird to me at first read. Would "was updated" read better?